### ISSUE TITLE
model.prediction() with argmax

### ISSUE THREAD

**Issue by MilesMena** [MilesMena/phenobench#3]
currently model.prediction() returns self.softmax(self.forward(x). so the torch.tensor([.98, .84, .16]) that we were seeing is not what we think it is.
